lib/setopt.c

```diff
@@ -2198,7 +2198,7 @@ CURLcode Curl_vsetopt(struct Curl_easy *data, CURLoption option, va_list param)
     else if(arg < UPLOADBUFFER_MIN)
       arg = UPLOADBUFFER_MIN;
 
-    data->set.upload_buffer_size = arg;
+    data->set.upload_buffer_size = (int)arg;
```

Comment:
Shouldn't this be casted to a uint? Even if a wrap is eliminated on line 2198 this would still look more natural to me

Reply:
Yes it should, thanks!